src/Symfony/Bundle/FrameworkBundle/CacheWarmer/TemplateFinder.php

```diff
@@ -36,7 +36,7 @@ class TemplateFinder implements TemplateFinderInterface
     /**
      * @param string $rootDir The directory where global templates can be stored
```

Comment:
type inconsistent with constructor definition
```suggestion
      * @param string|null $rootDir The directory where global templates can be stored
```